azure-cli/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+if [ -n "$SCRIPT_PATH" ]
+then
+  chmod +x ${GITHUB_WORKSPACE}/${SCRIPT_PATH}
```

Comment:
Should we check if the file exists and throw a better message if that is not the case

Reply:
done